File: java/236A.java
// 236 A. Boy or Girl

import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.next();
        Set<Character> unique = new HashSet<>();

        for (char c : s.toCharArray())
            unique.add(c);

       if(unique.size() % 2 == 0) System.out.println("CHAT WITH HER!");
       else System.out.println("IGNORE HIM!");

       sc.close();
    }
}


// testcase
// wjmzbmr
// sevenkplus
// xiaodao

File: cpp/236A.cpp
// 236 A. Boy or Girl

#include <bits/stdc++.h>
using namespace std;

int main() {
    string s;
    cin >> s;
    set<char> unique;

    for (char c : s)
        unique.insert(c);

    if (unique.size() % 2 == 0)
        cout << "CHAT WITH HER!" << endl;
    else
        cout << "IGNORE HIM!" << endl;

    return 0;
}


// testcase
// wjmzbmr
// sevenkplus
// xiaodao